src/main/java/uk/gov/cshr/controller/signup/SignupController.java

```diff
@@ -93,14 +99,24 @@ public String sendInvite(Model model,
         }
 
         final String email = form.getEmail();
-
-        if (inviteRepository.existsByForEmailAndStatus(email, InviteStatus.PENDING)) {
+        Optional<Invite> invite = inviteService.findByForEmailAndStatus(email, InviteStatus.PENDING);
+        if(invite.isPresent()) {
+            if (!inviteService.ifExpired(invite.get())) {
```

Comment:
Would it be possible to make positive if condition rather than "Not"?

Reply:
Done